src/locales/de.yaml

```diff
@@ -7,40 +7,40 @@ app:
       home: Dashboard
       add_printer: Drucker hinzufügen
       endstops: Endschalter
-      fans_outputs: Fans & Ausgänge
+      fans_outputs: Lüfter & Ausgänge
       jobs: Aufträge
-      macros: Macros
+      macros: Makros
       limits: Drucker Limits
       logs: Drucker Logs
       system_control: Systemsteuerung
-      runout_sensors: Runout Sensors
+      runout_sensors: Runout Sensoren
       status: Status
       temperature: Temperaturen
-      tool: Tool
+      tool: Druckkopf
       tune: Tune
       configure: Konfiguration
       interface: Interface
-      stats: Drucker Status
+      stats: Druckerstatus
       history: Auftragsverlauf
-      config_files: Konfigurations Dateien
+      config_files: Konfigurationsdateien
     label:
-      accel_to_decel: Accel zu Decel
-      acceleration: Acceleration
-      sqv: Square Corner Velocity
-      velocity: Velocity
-      api_url: API URL
+      accel_to_decel: Beschleunigen zu Bremsen
+      acceleration: Beschleunigung
+      sqv: Geschwindigkeit bei rechtwinkligen Ecken
+      velocity: Geschwindigkeit
+      api_url: API-URL
       host: Host
       layout: Layout
       name: Name
-      power: Power
+      power: Leistung
       printers: Drucker
       requested_speed: Gewünschte Geschwindigkeit
       services: Services
       speed: Geschwindigkeit
       flow: Flow
       z_offset: Z Offset
-      high: High
-      low: Low
+      high: Hoch
+      low: Niedrig
```

Comment:
Are these limits? if so `Maximum` and `Minimum` would work better here

Reply:
They aren't limits, high and low are used where you have a sensor to show the known high and low values. Ie., rolling over the pi cpu temp sensor would give you the current, high and low temps.

Reply:
I changed it to translate to "highest" and "lowest". I hope that fits.